ibr/ibr.proto

```diff
@@ -78,21 +122,25 @@ message InternalBuildingRepresentation {
   // The compressed coordinates that is referenced throughout the message.
   // A base64 encoded ordered list of coordinates.
   // The decoded string comes in the form of comma separated coordinates which
-  // are space separated floats. e.g. "<x> <y>,<x> <y>,<x> <y>".
-  // Visualization and other layers index into coordinates_lookup to determine coordinate
-  // segments.
+  // are space separated floats. e.g. "<x> <y> <z>,<x> <y> <z>,<x> <y> <z>".
+  // Visualization and other layers index into coordinates_lookup to determine
+  // coordinate segments.
   bytes coordinates_lookup = 5;
 
   // A contained layer/visualization data (e.g. tri gross area layer,
   // external/internal walls, blocking geometry etc.)
   repeated Visualization visualization = 6;
-  
-  // Boundary of the structure
+
+  // Boundary of the structure, coordinates stored in lookup.
```

Comment:
I think we need to define this more.  It needs to either be the beginning and ending of the indexes in the coordinates lookup, or we need to have this be another coordinates_lookup that is only a 2D vector.

Reply:
hm yeah, or could we just store this as it's own coordinates_lookup (x, y, x, y)?

Reply:
@danielmi-google  can we land this and iterate when we decide if boundary needs to change?